documentation/Logical Volume Manager (LVM) format.asciidoc

```diff
@@ -203,7 +205,7 @@ The offset, in bytes, relative from the start of the metadata area
 | 8 | 8 | | Data area size +
 Value in bytes
 | 16 | 4 | | Checksum +
-CRC-32 of [yellow-background]*TODO (metadata?)*
+CRC-32 for the data area
```

Comment:
which range ?

Reply:
Finally had some time to look into this, its a weak CRC-32 (CRC without XOR with 0xffffffff)
using the polynominal 0xedb88320 and initial value 0xf597a6cf